lib/Generator.php

```diff
@@ -52,7 +52,12 @@ public function __construct(PasswordPolicyConfig $config,
 	 * @throws HintException
 	 */
 	public function generate(): string {
-		$lenght = $this->config->getMinLength();
+		$minLength = $this->config->getMinLength();
+		if ($minLength < 8) {
+			// 8 minimum so we don't generate too short passwords
+			$minLength = 8;
+		}
+		$lenght = $minLength;
```

Comment:
`lenght` / `length` :see_no_evil:

Reply:
I hesitated to fix the original typo. maybe the time has come to do this :superhero: